sweep/sweeper.go

```diff
@@ -59,20 +59,30 @@ func (s *Sweeper) CreateSweepTx(
 	}
 
 	signDesc := lndclient.SignDescriptor{
-		WitnessScript: htlc.Script(),
+		WitnessScript: witnessScript,
 		Output: &wire.TxOut{
 			Value:    int64(amount),
 			PkScript: htlc.PkScript,
 		},
-		HashType:   txscript.SigHashAll,
+		HashType:   htlc.SigHash(),
 		InputIndex: 0,
 		KeyDesc: keychain.KeyDescriptor{
 			PubKey: key,
 		},
 	}
 
+	// We need our previous outputs for taproot spends, and there's no
+	// harm including them for segwit v0, so we always inlucde our prevOut.
```

Comment:
s/inlucde/include

This is definitely my typo, I can feel it :')

Reply:
fixed